MultimodalQnA/ui/gradio/utils.py

```diff
@@ -186,3 +190,15 @@ def convert_audio_to_base64(audio_path):
     """Convert .wav file to base64 string."""
     encoded_string = base64.b64encode(open(audio_path, "rb").read())
     return encoded_string.decode("utf-8")
+
+def convert_base64_to_audio(b64_str):
+    """Decodes the base64 encoded audio data and returns a saved filepath."""
+    
+    audio_data = base64.b64decode(b64_str)
+
+    # Create a temporary file
+    with tempfile.NamedTemporaryFile(delete=False, suffix=".wav") as temp_audio:
```

Comment:
Like, can you put `dir=TMP_DIR` in here and then put `TMP_DIR="/tmp"` at the top of the file?